build/dockerfiles/rhel.install.sh

```diff
@@ -7,8 +7,16 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
+arch="$(uname -m)" && baseurl="http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/${arch}/" && if [[ ${arch} != "x86_64" ]]; then baseurl="https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/${arch}/os/"; fi;\
+microdnf install -y findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
 
-microdnf install -y findutils bash wget yum gzip git tar jq python3-six python3-pip skopeo && microdnf -y clean all && \
+echo "[fedora]
```

Comment:
we have some repo files there https://github.com/eclipse/che-devfile-registry/blob/master/build/dockerfiles/content_sets_centos8_appstream.repo

so maybe use external file as well, but I let @nickboldt decide

Reply:
Good point, @benoitf  -- an external .repo file is easier to exclude from the Dockerfile when we transform it into the brew-friendly version
* upstream: https://github.com/eclipse/che-plugin-registry/blob/master/build/dockerfiles/rhel.Dockerfile#L45
* downstream: comment out that line; we don't use .repo files as everything is resolved from inside Brew/OSBS

Reply:
So... two ways we can avoid having to have multiple arch-specific .repo files:

a) put all arches in the same file, then assume yum/dnf will only install for the compatible arch; requires hardcoding of the supported arches:

```

[fedora_x86_64]
name=Fedora_x86_64
baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/x86_64/
gpgcheck=0
enabled=0
skip_if_unavailable=True

[fedora_s390x]
name=Fedora_s390x
baseurl=https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/s390x/os/
gpgcheck=0
enabled=0
skip_if_unavailable=True

[fedora_ppc64le]
name=Fedora_ppc64le
baseurl=https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/ppc64le/os/
gpgcheck=0
enabled=0
skip_if_unavailable=True
```

```
#➔ yum search skopeo 
skopeo.x86_64 : Inspect container images and repositories on registries
skopeo.s390x : Inspect Docker images and repositories on registries
skopeo.ppc64le : Inspect Docker images and repositories on registries

Installing:
 skopeo                                     x86_64                                     1:0.2.0-1.fc31                                       updates                                     6.5 M

Installed:
  skopeo-1:0.2.0-1.fc31.x86_64                                                  
```

b) put both URLs in the same file, and let the invalid one throw a harmless 404 error. 

```
[fedora30-updates]
name=Fedora 30 updates - $basearch
baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/$basearch/
gpgcheck=0
enabled=1
skip_if_unavailable=True

[fedora30-secondary-updates]
name=Fedora 30 secondary updates - $basearch
baseurl=https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/$basearch/os/
gpgcheck=0
enabled=1
skip_if_unavailable=True
```

Same as above but only one platform is presented to yum, and when reading repos, an ignored 404 is thrown:

```
#➔ yum search skopeo 
skopeo.x86_64 : Inspect container images and repositories on registries
```

```
Errors during downloading metadata for repository 'fedora30-secondary-updates':
  - Status code: 404 for https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/x86_64/os/repodata/repomd.xml (IP: 209.132.181.23)
Error: Failed to download metadata for repo 'fedora30-secondary-updates': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
Ignoring repositories: fedora30-secondary-updates
```

I'm inclined to go with the 2nd option. WDYT?

Reply:
@nickboldt Second option you mentioned above is working for us.

Reply:
* is the file readable after copying to /etc/yum.repos.d/fedora.repo? 

* if `/usr/bin/pip3.6` isn't found it's either because the symlink is not created (or python and pip 3.7 is installed instead, perhaps because you have access to Fedora and RHEL repos)

Can you share more of the build log / error report? Can you add the .repo file to this PR?

Reply:
@nickboldt I have made following changes to the files and now it is working. Please check.

1. Created ```build/dockerfiles/fedora.repo``` with following content
```
[fedora30-updates]
name=Fedora 30 updates - $basearch
baseurl=http://download.fedoraproject.org/pub/fedora/linux/updates/30/Everything/$basearch/
gpgcheck=0
enabled=1
skip_if_unavailable=True

[fedora30-secondary-updates]
name=Fedora 30 secondary updates - $basearch
baseurl=https://dl.fedoraproject.org/pub/fedora-secondary/releases/30/Everything/$basearch/os/
gpgcheck=0
enabled=1
skip_if_unavailable=True
````
2.Added below line in ```build/dockerfiles/rhel.Dockerfile```
```
COPY ./build/dockerfiles/fedora.repo /etc/yum.repos.d/
```
3. Added below lines in ```build/dockerfiles/rhel.install.sh```
```
microdnf --disablerepo=fedora30-updates --disablerepo=fedora30-secondary-updates install -y findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
microdnf --enablerepo=fedora30-updates --enablerepo=fedora30-secondary-updates install -y skopeo jq && microdnf update -y skopeo containers-common jq oniguruma && microdnf -y clean all; \
```

Reply:
looks reasonable. Can you merge these changes into your PR so I can see the whole completed result?

Reply:
@nickboldt Merged above code changes in PR. Please review.